app/views/dashboard/_past_applications.haml

```diff
@@ -1,17 +1,21 @@
-.dashboard_partial
-  .date_range
-    = form_tag past_applications_application_submissions_path, method: :get do
-      List applications for
-      .note_item
-        = select_tag :department_ids,
+.dashboard_partial.container
```

Comment:
`.container` at this level makes this box narrower than the rest of the ones in the dashboard (because of added margin).